bcgov/common-hosted-form-service
# forms-1226-Team management functionality && Fix for failing tests on Pr env

# Description

1. Fix for failing tests on PR env 
2. Team Management functionality of CHEFS before and after form publish





## Test

1. Create a form and add team members ,roles and verify the roles and visibility in the dashboard
2. Remove the user
3. Publish the form and add the team members 
4. Delete the form after each cypress test run








## Checklist




- [x] I have added tests that checks the functionalities of CHEFS frond end
- [x] I have approval from the product owner for the contribution in this pull request

## Further comments
